lib/galaxy/jobs/runners/util/job_script/__init__.py

```diff
@@ -135,8 +139,11 @@ def _handle_script_integrity(path, config):
             except Exception:
                 pass
             time.sleep(sleep_amt)
-        except Exception:
-            pass
+        except OSError:
```

Comment:
This would still fail the check, I think you'd need to pull this into the for loop.

Reply:
it would? My reading is that it is in the for loop and those exceptions which we now log would just provide information on every execution. I've just woken up so maybe I'm missing something?